Pages/Feedback.cshtml

```diff
@@ -28,23 +28,23 @@
                 Let us know about your neighborhood
             </h2>
         </div>
-        <div class="">
+        <div class="container">
```

Comment:
Is the class container used in css?

Avoid adding unimplemented features, as unused code is clutter, which is technical debt.